### PR TITLE
Remove logs and report files from targets' inputs

### PR DESCRIPTION
**Pull request by eszpotanski**
This PR adds filtering of targets' inputs to not depend on log and report files.

### CODE REVIEW

File: openroad.bzl [hunk 1]
```diff
@@ -297,6 +297,19 @@ def get_entrypoint_cmd(
...
+    Filters out log and report files from stage's output.
+
+    Args:
+      outs: dictionary of lists with paths to output files for each flow stage
+      stage: name of the stage
+
+    Retruns:
+      list with filtered output files
+    """
+    return [i for i in outs.get(stage, []) if len(i) > 4 and i[-4:] not in (".log", ".rpt")]
```
Comment (hovind) [inline openroad.bzl:311]: Should we use https://bazel.build/rules/lib/core/string#endswith?

> sub 	[string](https://bazel.build/rules/lib/core/string.html); or [tuple](https://bazel.build/rules/lib/core/tuple.html) of [string](https://bazel.build/rules/lib/core/string.html)s; required
The suffix (or tuple of alternative suffixes) to match.

**Review by hovind — APPROVED** [review]
Seems reasonable to me, nice!

File: openroad.bzl (hunk 1 above)
Comment (oharboe) [reply openroad.bzl:311]: endswith doesn't jell with "in (...)", so it would have to be map... Good 'nuf for now.

**Comment by oharboe** [comment]
@eszpotanski Did a squash and merge this time. I suppose it was the intention to squash these two commits, but it wasn't done for some reason?

**Pull request merged**